Twitch/Twitch.Base/Services/NewAPI/ChannelsService.cs

```diff
@@ -83,6 +83,25 @@ public async Task<IEnumerable<ChannelBannedEventModel>> GetChannelBannedEvents(U
             return await this.GetPagedDataResultAsync<ChannelBannedEventModel>("moderation/banned/events?broadcaster_id=" + channel.id, maxResults);
         }
 
+
+        /// <summary>
+        /// Returns all banned and timed-out users in a channel.
+        /// </summary>
+        /// <param name="channel">The channel to get banned events for</param>
+        /// <param name="userIDs">If specified, filters banned and timed-out users to those userIDs specified.</param>
+        /// <param name="maxResults">The maximum number of results. Will be either that amount or slightly more</param>
+        /// <returns>The set of banned or timed-out users</returns>
+        public async Task<IEnumerable<ChannelBannedUserModel>> GetChannelBannedUsers(UserModel channel, IEnumerable<string> userIDs=null, int maxResults=1)
+        {
+            Validator.ValidateVariable(channel, "channel");
+            List<string> parameters = new List<string>();
+            foreach (string userID in userIDs)
```

Comment:
Since userIDs can be null, we need to add a null check here before looping through it

Reply:
I added a null check but I am not sure it is really needed.  If the value is null then the foreach never loops.  I tested that earlier today and it seemed to work fine without any exception.